content/media/gigascience/convert-images.sh

```diff
@@ -0,0 +1,28 @@
+
+# Figure 1
+# TODO export content/media/rephetio/metagraph-and-features.psd to EPS via photo
```

Comment:
@vincerubinetti if you still have photoshop, is it possible to export [`metagraph-and-features.psd`](https://github.com/dhimmel/connectivity-search-manuscript/blob/c560f51adfb1a3113b7cb1419a294cb80be2749a/content/media/rephetio/metagraph-and-features.psd) to vector formats? EPS for Gigascience, and then PDF & SVG if they're easy just so we have them. I think the source for Figure 1 in Photoshop is vector, but correct me if its raster and it doesn't make sense to export vectors.

Reply:
That PSD file is just a basic import of the two figures and adds the lettering. But each individual pane is from an SVG source. I can reassemble them in Illustrator and replace the PSD.

Reply:
Ah, so the photoshop isn't suitable for the vector export (or already has rasterized things)? I'd say if it takes <= 45 minutes, it's worth the benefits here, otherwise likely not.

Reply:
Photoshop can do vector export I think, but it only makes sense if the layers you have in the photoshop file are vector format. In this case, I think I imported them as high res raster because when I wrote the feature matrix svg by hand, I used things like font imports and reusable symbols to save time, but which svg editors like Illustrator and Inkscape don't understand.

As such, I've made "rendered" versions of both the metagraph svg and feature matrix svg. By "rendered", I mean, they're still svgs, but the advanced features like clipping/fonts/symbols have been "flattened" into simple paths that should be properly viewable by any svg software (but not as easily editable). I did this by opening them in Chrome, printing to pdf, importing the pdf into Illustrator, saving with the "convert text to outlines" option, and doing a bit of manual cleanup with SVGOMG.

Files: [figures.zip](https://github.com/greenelab/connectivity-search-manuscript/files/11850952/figures.zip)

We may want to put the rendered metagraph in the het.io repo. And you can delete the .psd here and replace it with the .ai file.

Reply:
Awesome! Uploaded the contents of figures in https://github.com/greenelab/connectivity-search-manuscript/pull/54/commits/8e428c3f18ad82df99df35828cd4bdc4e7a80111 and replaced the .psd with the .ai file.

Will also upload `metagraph-rendered.svg` to the [`het.io`](https://github.com/hetio/het.io/tree/683bc8b4654faeca863ff21b2e2fd4dde5c89f0c/about) repo.